CONTRIBUTING.md

```diff
@@ -1,66 +1,59 @@
-Contributing to StormCrawler
+Contributing to Apache StormCrawler
 ============================
 
+Thank you for your intention to contribute to Apache StormCrawler. As an open-source community, we highly appreciate external contributions to our project.
```

Comment:
do we need 'external' here, who is considered internal then?

Reply:
Let's drop external :-)

Reply:
Fixed @ayushtkn - thx for review!